docs/resource/process.md

```diff
@@ -27,32 +28,40 @@
+| [`process.executable.name`](../attributes-registry/process.md) | string | The name of the process executable. On Linux based systems, can be set to the `Name` in `proc/[pid]/status`. On Windows, can be set to the base name of `GetProcessImageFileNameW`. | `otelcol` | `Conditionally Required` [4] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`process.executable.path`](../attributes-registry/process.md) | string | The full path to the process executable. On Linux based systems, can be set to the target of `proc/[pid]/exe`. On Windows, can be set to the result of `GetProcessImageFileNameW`. | `/usr/bin/cmd/otelcol` | `Conditionally Required` [5] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`process.owner`](../attributes-registry/process.md) | string | The username of the user that owns the process. | `root` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`process.parent_pid`](../attributes-registry/process.md) | int | Parent Process identifier (PPID). | `111` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`process.pid`](../attributes-registry/process.md) | int | Process identifier (PID). | `1234` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
-**Additional attribute requirements:** At least one of the following sets of attributes is required:
+**[1]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[2]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[3]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[4]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+
+**[5]:** See [Selecting process attributes](#selecting-process-attributes) for details.
+<!-- endsemconv -->
+
+### Selecting process attributes
+
+Instrumentations MUST populate at least one of the following attributes:
 
 * [`process.executable.name`](../attributes-registry/process.md)
 * [`process.executable.path`](../attributes-registry/process.md)
 * [`process.command`](../attributes-registry/process.md)
 * [`process.command_line`](../attributes-registry/process.md)
 * [`process.command_args`](../attributes-registry/process.md)
-<!-- endsemconv -->
 
 Between `process.command_args` and `process.command_line`, usually `process.command_args` should be preferred.
 On Windows and other systems where the native format of process commands is a single string,
 `process.command_line` can additionally (or instead) be used.
 
-For backwards compatibility with older versions of this semantic convention,
```

Comment:
This note has been added in https://github.com/open-telemetry/opentelemetry-specification/pull/1137, spec version v0.7.0.
I think we can forget that `process.command_line` used to have array type 3.5 years ago.